docs/spec/terminology.md

```diff
@@ -17,8 +17,9 @@
 | Multihash  { #multihash }            | Protocol for differentiating outputs from common cryptographic hash functions, addressing size + encoding considerations: https://multiformats.io/multihash/ |
 | DID Operation         | Set of delta-based modifications that change the state of a DID Document when applied.                                               |
 | Operation Request     | JWS formatted request sent to a _Sidetree Node_ to include a _DID Operation_ in a batch of operations.     |
+| Operation Key     | A cryptographic key used to produce an _Operation Request_ JWS. MAY be present in the DID Document. MUST be used to produce _Operation Request_ commitment.     |
+| Recovery Key          | A cryptographic key used to produce an _Operation Request_ of type Recover or Deactivate. MAY be present in the DID Document. MUST be used to produce _Operation Request_ commitment.         |
 | Public Key Commitment { #public-key-commitment } | The resulting [commitment](#commitment) obtained by applying the defined [commitment scheme](#operation-commitment-scheme) to a public key |
-| Recovery Key Pair { #recovery-key-pair }    | Public/private key pair used to perform a Recovery or Deactivate operation.          |
```

Comment:
Can we check and change the usages of this removed definition throughout the spec?

Reply:
feels like we should use the term "Key Pair" everywhere... and maybe define "Key Pair" as its own thing as well.

Reply:
@tplooker I believe I have addressed this.